### PR TITLE
Fix/jwt verify secret

### PR DESCRIPTION
**Pull request by ThijsZijdel**
- [x] Fix initial verify
- [ ] Implement the algorithms
- [ ] Nicer output of the verify

### CODE REVIEW

**Comment by ThijsZijdel** [comment]
Screenshot 2024-08-06 at 16 24 53

Screenshot 2024-08-06 at 16 25 00

Screenshot 2024-08-06 at 16 25 06

File: src/Features/jwt/JWT.tsx
Comment (ThijsZijdel) [inline src/Features/jwt/JWT.tsx:1]: Initial verify is setup correctly now.

**Comment by Sparkenstein** [comment]
I think both the open PRs #84 and #86  have repeated changes from merged PR #85. 

Can you create a new PRs with only features that you want to merge.  This one can have only JWT verify signature part, so pull upstream that should solve the repeated code

**Review by Sparkenstein — APPROVED** [review]
LGTM. thank you for the feature 👍

**Pull request merged**